cli/pcluster/config/validators.py

```diff
@@ -404,11 +404,15 @@ def efa_validator(param_key, param_value, pcluster_config):
     cluster_section = pcluster_config.get_section("cluster")
 
     allowed_instances = _get_efa_enabled_instance_types(errors)
-    if cluster_section.get_param_value("compute_instance_type") not in allowed_instances:
-        errors.append(
-            "When using 'enable_efa = {0}' it is required to set the 'compute_instance_type' parameter "
-            "to one of the following values : {1}".format(param_value, allowed_instances)
-        )
+    if pcluster_config.cluster_model.name == "SIT":
+        # Specific validations for SIT clusters
+        if cluster_section.get_param_value("compute_instance_type") not in allowed_instances:
+            errors.append(
+                "When using 'enable_efa = {0}' it is required to set the 'compute_instance_type' parameter "
+                "to one of the following values : {1}".format(param_value, allowed_instances)
+            )
+        if cluster_section.get_param_value("placement_group") is None:
+            warnings.append("You may see better performance using a cluster placement group.")
```

Comment:
What about a link to the official placement group docuemntation

Reply:
I wouldn't change this in this patch, as the message comes frome previous versions.

Reply:
ok